packages/near-api-js/src/account.ts

```diff
@@ -482,7 +482,26 @@ export class Account {
      * @param options.blockQuery specifies which block to query state at. By default returns last "optimistic" block (i.e. not necessarily finalized).
      * @returns {Promise<any>}
      */
-    async viewFunction(
```

Comment:
I think the best approach here may be to create a `viewFunctionV2` with the new functionality and always call that similar to [this change set](https://github.com/near/near-api-js/commit/a05667361ddab208423f5a5c0e3dd0ce182b9880#diff-ae151444b91b7e5e0a6aa753024f7900f0ff0447afaa2d18ad6619d5c3e12318L156-R224) when signAndSendTransactions was converted to having an object argument. We likely should deprecate the current way of calling `viewFunction` too.  cc @janewang